examples/cpp/20_native_dpcpp_kernel/addVectors.cpp

```diff
@@ -0,0 +1,12 @@
+#include <CL/sycl.hpp>
+
+extern "C" void addVectors_it(::sycl::queue *q, ::sycl::nd_range<3> *ndrange, int &entries, int *oa, int *ob, int *oc)
```

Comment:
Can we keep the variables consistent with the other examples? Splitting the arguments in new lines can also make the expected arguments + ordering more readable

```suggestion
extern "C" void addVectors_it(::sycl::queue *q,
                              ::sycl::nd_range<3> *ndrange,
                              int &entries,
                              int *a,
                              int *b,
                              int *ab) {
```